cocos/2d/assets/sprite-frame.ts

```diff
@@ -27,10 +27,11 @@
 
 import { ccclass } from 'cc.decorator';
 import { EDITOR, TEST, BUILD } from 'internal:constants';
+import { ImageSource } from 'pal/image/types';
```

Comment:
please use relative path, we don't have module called 'pal/image/types'

this is an absolute path, which is not encouraged

Reply:
This is an error, it should be referencing 'pal/image'.